announcement/announcement-impl/impl/src/java/org/sakaiproject/announcement/impl/SiteEmailNotificationAnnc.java

```diff
@@ -509,21 +509,22 @@ protected String plainTextContent(Event event) {
 		{
 			Site site = siteService.getSite(siteId);
 			title = site.getTitle();
+			url = site.getUrl(); // Might have a better URL.
 		}
 		catch (Exception ignore)
```

Comment:
as this branch has been already merged I have made changes in the new pull request.